packages/cli/src/zosfiles/compare/sdd/Spooldd.handler.ts

```diff
@@ -0,0 +1,79 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+import { AbstractSession, IHandlerParameters, ITaskWithStatus, TaskStage, DiffUtils } from "@zowe/imperative";
+import { IZosFilesResponse } from "@zowe/zos-files-for-zowe-sdk";
+import { GetJobs } from "@zowe/zos-jobs-for-zowe-sdk";
+import { ZosFilesBaseHandler } from "../../ZosFilesBase.handler";
+/**
+ * Handler to compare spooldd's content
+ * @export
+ */
+export default class SpoolddHandler extends ZosFilesBaseHandler {
+    public async processWithSession(commandParameters: IHandlerParameters, session: AbstractSession): Promise<IZosFilesResponse> {
+        const task: ITaskWithStatus = {
+            percentComplete: 0,
+            statusMessage: "Retrieving first spool dd",
+            stageName: TaskStage.IN_PROGRESS
+        };
+        const descriptionSeperator: string = ":";
+        commandParameters.response.progress.startBar({ task });
+        const spoolDescription1 = commandParameters.arguments.spoolDescription1;
+
+        // retrieving information for first spooldd
+        const spoolDescArr1 = spoolDescription1.split(descriptionSeperator);
+        const jobName1: string = spoolDescArr1[0];
+        const jobId1: string = spoolDescArr1[1];
+        const spoolId1: number = Number(spoolDescArr1[2]);
```

Comment:
thoughts on a utility function for this ?
Maybe we can simplify this to `const [jobName1, jobId1, SpoolId1] = getSpoolDescriptor(spoolDescription1)`

Reply:
Should it be in the same handler :thinking:?

Reply:
No need to do this.
However, if we do it, I think it should be in:
- the zos-files-sdk, or
- the core-sdk (so that in can be shared between the files and jobs SDKs)